pkg/solana/logpoller/models.go

```diff
@@ -26,16 +26,9 @@ func (f Filter) MatchSameLogs(other Filter) bool {
 		f.EventIdl.Equal(other.EventIdl) && f.SubkeyPaths.Equal(other.SubkeyPaths)
 }
 
-// Discriminator returns a 12 character base64-encoded string
-//
-// This is the base64 encoding of the [8]byte discriminator returned by utils.Discriminator
-func (f Filter) Discriminator() string {
-	d := Discriminator("event", f.EventName)
-	b64encoded := base64.StdEncoding.EncodeToString(d[:])
-	if len(b64encoded) != 12 {
-		panic(fmt.Sprintf("Assumption Violation: expected encoding/base64 to return 12 character base64-encoding, got %d characters", len(b64encoded)))
-	}
-	return b64encoded
+// DiscriminatorRawBytes returns raw discriminator bytes as a string, this string is not base64 encoded.
```

Comment:
I think it would be worth mentioning here that this is guaranteed to return a string of length 8, in case someone calling it is unsure whether they need to validate what's returned.